test/jit/test_hooks.py

```diff
@@ -158,6 +179,7 @@ def forward_hook(self, input: Tuple[str], output: str):
         self.assertEqual(m.forward("a"), m_scripted.forward("a"))
         self.assertNotEqual(m_scripted("a"), m_scripted.forward("a"))
 
+    @unittest.skipIf(IS_SANDCASTLE or IS_FBCODE, saving_msg)
```

Comment:
As per Python documentation, you can skip the entire class using the same decorator instead of each method separately:
```
@unittest.skip("showing class skipping")
class MySkippedTestCase(unittest.TestCase):
    def test_not_run(self):
        pass
```

Reply:
There are a few tests that can run still, the hook error messages. I wasn't sure if I should skip those or not

Reply:
Oops, forgot about those.